locales/en-US/messages.properties

```diff
@@ -135,10 +135,10 @@ publishedLink=Published Version
 yourProjectsLink=Your Projects
 loginWelcome=Hi, {{username}}
 newProjectLink=New Project
-signOutLink=Sign out
+signOutLink2=Sign out
```

Comment:
(actually, should write that here…)
This one does not need to be updated (capitalization rules are specific to each language, so translations are already correct)